dpdata/gaussian/gjf.py

```diff
@@ -0,0 +1,36 @@
+def to_gjf_string(system, frame_idx=0, header="", title="", foot="", charge=0, mult=1):
```

Comment:
DP-GEN has a more complex function at https://github.com/deepmodeling/dpgen/blob/00432d2349039a3ee07817426b9aa4d7fce0f7b6/dpgen/generator/lib/gaussian.py#L98-L185. Can you take a look?

Reply:
According to @amcadmus 's advice, I have removed support for gaussian input files. We may later discuss how to integrate the similar function implemented in DP-GEN to dpdata. @njzjz